src/schemas/__init__.py

```
@@ -21,6 +21,12 @@
     CartItemResponse,
     CartResponse
 )
+
+from schemas.orders import (
+    OrderItemResponse,
```

Ruff (0.11.9)

`schemas.orders.OrderItemResponse` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)